app/controllers/admin/invisiblizes_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Admin
+  class InvisiblizesController < ApplicationController
+    def create
+      developer.invisible!
+      redirect_to developers_path, notice: t(".created")
+    end
+
+    private
+
+    def developer
+      @developer ||= Developer.find(params[:developer_id])
+    end
```

Comment:
Let's move this to line 4 as `Developer.find(params[:developer_id]).invisible!`.